src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -68,7 +72,25 @@ define(function (require, exports, module) {
             /**
              * @type {Boolean}
              */
-            disabled: React.PropTypes.bool
+            disabled: React.PropTypes.bool,
+
+            /**
+             * True if the LayerGroup should render collapsed layers. This is controlled by the 
+             * root LayerGroup.
+             * 
+             * @type {Boolean}
+             */
+            renderCollapsedLayers: React.PropTypes.bool
+        },
+        
+        componentDidMount: function () {
+            if (!this.props.parentLayer) {
+                this.renderCollapsedLayers = true;
+
+                window.setTimeout(function () {
+                    this.forceUpdate.bind(this, null);
```

Comment:
Wait, what? I have no idea what's going on here. You're binding a function but then throwing it away after a 2 second timeout? I'm confused!